primo-explore/custom/main.js

```diff
@@ -515,8 +515,20 @@ app.controller('FullViewAfterController', ['angularLoad', '$http', '$scope', '$e
   {
     vm.parentElement = this.$element.parent()[0]; //the prm-full-view container
     try {
-        vm.doi = vm.parentCtrl.item.pnx.addata.doi[0] || '';
+      console.log(vm.parentCtrl.item.pnx.display);
+      console.log(vm.parentCtrl.item.pnx.addata);
```

Comment:
Quite a few `console.log` added. Can we remove some, or are they all required?

Reply:
Oh, I will remove those.